ports/stm/supervisor/usb.c

```diff
@@ -80,11 +85,13 @@ void init_usb_hardware(void) {
     never_reset_pin_number(0, 12);
 
     /* Configure VBUS Pin */
+#if (BOARD_NO_VBUS_SENSE == 0)
```

Comment:
I would suggest `#if !(BOARD_NO_VBUS_SENSE) as it is simply omitted for most boards and is better in line with existing flag style.